Libraries/Components/Picker/Picker.js

```diff
@@ -166,24 +166,22 @@ class Picker extends React.Component<PickerProps> {
 
   static Item: typeof PickerItem = PickerItem;
 
-  static defaultProps: {|mode: $TEMPORARY$string<'dialog'>|} = {
-    mode: MODE_DIALOG,
-  };
-
   render(): React.Node {
+    const {mode = MODE_DIALOG, ...rest} = this.props;
+
     if (Platform.OS === 'ios') {
       /* $FlowFixMe[prop-missing] (>=0.81.0 site=react_native_ios_fb) This
        * suppression was added when renaming suppression sites. */
       /* $FlowFixMe[incompatible-type] (>=0.81.0 site=react_native_ios_fb) This
        * suppression was added when renaming suppression sites. */
-      return <PickerIOS {...this.props}>{this.props.children}</PickerIOS>;
+      return <PickerIOS mode={mode} {...rest} />;
```

Comment:
Somewhat of a nit and open for discussion but I think it'd be clearer if we kept the children as the "inner child" of this component but I don't feel strongly. We can extract it from `this.props` on line 170 so you can still spread `rest`

Reply:
``` jsx
<PickerIOS mode={mode} {...rest}>
  {children}
</PickerIOS>
```

this way will appear flow errors:

![image](https://user-images.githubusercontent.com/33770367/120733446-c67f0700-c519-11eb-8093-924904123bb9.png)